busted-scm-1.rockspec

```diff
@@ -34,7 +34,7 @@ description = {
 
 dependencies = {
   'lua >= 5.1',
-  'lua_cliargs = 3.0',
+  'lua_cliargs >= 3.0',
```

Comment:
```suggestion
  'lua_cliargs ~> 3',
```
Wouldn't it be safer to pin to the major version?

Reply:
Not "safer" for sure but maybe "safe enough". If we follow semver specs breaking changes should only be in major versions. I think our big question is whether we're going to consider the change in clirags to `--help` handling so it doesn't throw an error exit status as a breaking change or not.

Reply:
I'd vote no...

Reply:
That was my first thought too, but we have a v4 coming up pretty soon to fix the SPLAT defaults, so if we *wanted* to hold off until it would be in a breaking release anyway we could. Personally I think the behavior is so *wrong* fixing it as a bug should be the right thing to do even if it does break a few apps expectations downstream.

Reply:
@alerque your call, I'm fine either way.